packages/webpack-cli/lib/groups/HelpGroup.js

```diff
@@ -115,6 +116,16 @@ class HelpGroup {
                     return e;
                 }),
             },
+            {
+                header: 'Negated Flags',
+                optionList: negatedFlags.map((e) => {
+                    e.description = `Negates ${e.name}`;
+                    e.name = `no-${e.name}`;
+                    e.alias = null;
+                    e.type = Boolean;
+                    return e;
+                }),
```

Comment:
I think it would be better to have It in the first place.

```js
const negatedFlags = options.core
            .filter((flag) => flag.negative)
            .reduce((allFlags, flag) => {
                return [...allFlags, { name: `no-${flag.name}`, description: `Negates ${flag.name}`, type: Boolean }];
            }, [])
            .map(({ alias, ...rest }) => rest); // remove alias or set it to null
```